schema/home/home_page_module.js

```diff
@@ -4,15 +4,32 @@ import Results from './results';
 import Title from './title';
 import Context from './context';
 import Params from './params';
+import { NodeInterface } from '../object_identification';
+import { toGlobalId } from 'graphql-relay';
 import {
   GraphQLObjectType,
   GraphQLString,
   GraphQLBoolean,
+  GraphQLID,
+  GraphQLNonNull,
 } from 'graphql';
 
-export const HomePageModulesType = new GraphQLObjectType({
-  name: 'HomePageModules',
+export const HomePageModuleType = new GraphQLObjectType({
+  name: 'HomePageModule',
+  interfaces: [NodeInterface],
```

Comment:
It’s the `NodeField` that handles the actual query: https://github.com/artsy/metaphysics/blob/e79d16d02efc26f110406dd9a0b10cc8b8cd490d/schema/object_identification.js#L94-L112

The inclusion of the interface here is merely there to express in the schema that this type conforms to the interface defined by `NodeInterface`, which in this case means it implements the `__id` field.